examples/persistence_transform.py

```diff
@@ -0,0 +1,41 @@
+"""
+========================================
+Persistence Transform in Mapcubes
+========================================
+
+Persistence Transform is a simple image processing technique that is useful for the visualization
+and depiction of gradually evolving structures.
+This example illustrates persistence transform using the AIA_193_CUTOUT series.
+"""
+import numpy as np
+import matplotlib.pyplot as plt
+import sunpy.map
+import sunpy.physics.differential_rotation as diffrot
+from sunpy.data.sample import AIA_193_CUTOUT01_IMAGE, AIA_193_CUTOUT02_IMAGE, AIA_193_CUTOUT03_IMAGE
+
+###############################################################################
+# We create the MapCube using the AIA_193_CUTOUT sample data.
+# To create a MapCube, we can call Map directly but add in a keyword to output a MapCube instead.
+aiamapcube = sunpy.map.Map(AIA_193_CUTOUT01_IMAGE, AIA_193_CUTOUT02_IMAGE,
+                           AIA_193_CUTOUT03_IMAGE, cube=True)
+
+###############################################################################
+# For a data set consisting of N images with intensity values I(x,y,t), the Persistence Map Pn is a
+# function of several arguments, namely intensity, location and time:
+# Pn(x,y,tn)=Q(I(x,y,t≤tn))
+persistence_maps = []
+persistence_maps.append(aiamapcube[0])    
+for i, map_i in enumerate(aiamapcube[1:]):
+    per = np.array([aiamapcube[n].data for n in [i, i+1]]).min(axis=0)
```

Comment:
The persistence transform as originally defined finds the maximum value achieved by a pixel for all times less than and including the current time.  So in the list comprehension above I think the index goes from [0, i] and you should use .max(axis=0)

Reply:
Ah, then with the current three images they won't be washed out... though probably the middle image will not affect much to the final result.